### ISSUE TITLE
Balanced asymmetric not respecting number of 'right' classes when 'left' is less than 'right' and central class is false

### ISSUE THREAD

**Issue by joewdavies** [riatelab/dicopal.js#7]
I think i might have found a bug:

Balanced asymmetric schemes appear to not respect the number of specified classes `right` when `left` is less than `right`:

For example here: there should be 4 classes to the right but there are only 3:

![image](https://github.com/riatelab/dicopal.js/assets/25485293/db8b59b3-5374-4f74-9615-d9502f594eec)

All the best!
Joe

**Comment by mthh** [comment]
Indeed.
But only when central class is false it seems (so I took the liberty to edit your issue title).

I do really need to refactor this code and add tests... Will do all of this this week I guess.

Thanks for the report !

**Comment by mthh** [comment]
This should be fixed by https://github.com/riatelab/dicopal.js/commit/1e2f76c02bf1a44df7024d950562282c54d1e808 but I want to strengthen the test suite a bit before making a new release.

**Comment by mthh** [comment]
I've released [0.7.0](https://github.com/riatelab/dicopal.js/releases/tag/0.7.0).

Thanks for all your reports.

And sorry you had to pay the piper a bit with dicopal! I've integrated dicopal into an app we're developing but the place where users can build their palettes isn't really functional yet (and we don't expect to release the application for a few months yet) so I haven't had a chance to run into all the bugs you ran into.
